public/data/kumar-kalyan.json

```diff
@@ -46,6 +46,13 @@
       "icon": "book",
       "color": "red",
       "description": "Started leading the technical communities as a student leader"
+    },
+    {
+      "title": "Became Google DSC Lead",
+      "date": "August 2022",
+      "icon": "book",
+      "color": "red",
+      "description": "As a chapter lead I am helping everyone to grow their knowledge in a peer-to-peer learning environment and build solutions for local businesses and communites"
```

Comment:
Minor spelling error in here.
```suggestion
      "description": "As a chapter lead I am helping everyone to grow their knowledge in a peer-to-peer learning environment and build solutions for local businesses and communities"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
Good spot 👍 . I think if the author doesn't respond after 1 week, we should merge their PR, as other changes are more important for their profile.

They can submit a new PR to fix the typo